robot/marker_setup/teams.py

```diff
@@ -4,10 +4,11 @@
 
 class TEAM(enum.Enum):
     # TODO: Think of better team names
-    LEON = "N6MAC41717"
-    ZHORA = "N6FAB61216"
-    PRIS = "N6FAB21416"
-    ROY = "N6MAA10816"
+    RUSSET = "N6MAC41717"
```

Comment:
Just wondering where these numbers come from

Reply:
they are left over easter eggs from last year, they are never referenced anywhere and have no effect on the code so ive left them in for now

Reply:
they can be removed if we want but they have no impact

Reply:
I know this isn't a problem which you have created but I think for the sake of readable code it would be good to just make these either:
```py
class TEAM(enum.Enum):
    RUSSET = 1
    SWEET = 2
```
or 
```py
class TEAM(enum.Enum):
     RUSSET = "RUSSET"
     SWEET = "SWEET"
```